server/service/src/sync/program_schemas/routine_immunisation_6weeks_encounter_ui_schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "type": "VerticalLayout",
+  "elements": [
+    {
+      "type": "Label",
+      "text": "Routine Immunisation 6 Weeks Encounter",
+      "variant": "h1"
+    },
+    {
+      "type": "Control",
+      "scope": "#/properties/vaccinator"
+    },
+    {
+      "type": "Control",
+      "scope": "#/properties/notes",
+      "options": { "width": "135px" }
+    },
+    {
+      "type": "Categorization",
+      "options": {
+        "variant": "tab"
+      },
+      "elements": [
+        {
+          "type": "Category",
+          "label": "Rota Virus",
+          "options": { "icon": "medicine" },
+          "elements": [
+            {
+              "type": "Control",
+              "scope": "#/properties/rotavirus/properties/location"
+            },
+            {
+              "type": "Control",
+              "scope": "#/properties/rotavirus/properties/vaccine"
+            }
+          ]
+        },
+        {
+          "type": "Category",
+          "label": "Diphtheria...",
+          "options": { "icon": "medicine" },
```

Comment:
another suggestion.. 

<img width="196" alt="Screenshot 2022-11-21 at 4 10 04 PM" src="https://user-images.githubusercontent.com/9192912/202955127-8bd76e14-70f3-4bff-bc1a-7a416ef00cef.png">

added in a new svg and updated in this branch: https://github.com/openmsupply/open-msupply/compare/immunisation-program-demo...immunisation-program-demo-icon?expand=1

Reply:
Awesome!